File: CuartoPunto/fibonacci.c
#include <stdio.h>
#include <time.h>

int fibonacci(int n) {
    if (n <= 1) return n;
    return fibonacci(n - 1) + fibonacci(n - 2);
}

int main() {
    int n = 40;
    clock_t inicio = clock();

    int resultado = fibonacci(n);

    clock_t fin = clock();
    double tiempo = (double)(fin - inicio) / CLOCKS_PER_SEC;

    printf("fibonacci(%d) = %d\n", n, resultado);
    printf("Tiempo: %.4f segundos\n", tiempo);
    return 0;
}

File: CuartoPunto/fibonacci.py
import time

def fibonacci(n):
    if n <= 1:
        return n
    return fibonacci(n - 1) + fibonacci(n - 2)

n = 40
inicio = time.time()

resultado = fibonacci(n)

fin = time.time()
tiempo = fin - inicio

print(f"fibonacci({n}) = {resultado}")
print(f"Tiempo: {tiempo:.4f} segundos")
